verticapy/learn/cluster.py

```diff
@@ -324,53 +299,36 @@ def drop_temp_elem(cursor):
                         clusters[node] = clusters[node_neighbor]
                 del graph[0]
             try:
-                f = open("VERTICAPY_DBSCAN_CLUSTERS_ID.csv", "w")
+                f = open("{}.csv".format(name_dbscan_clusters), "w")
                 for elem in clusters:
                     f.write("{}, {}\n".format(elem, clusters[elem]))
                 f.close()
-                try:
-                    cursor.execute(
-                        "DROP TABLE IF EXISTS v_temp_schema.VERTICAPY_DBSCAN_CLUSTERS"
-                    )
-                except:
-                    pass
-                cursor.execute(
-                    "CREATE LOCAL TEMPORARY TABLE VERTICAPY_DBSCAN_CLUSTERS(node_id int, cluster int) ON COMMIT PRESERVE ROWS"
-                )
-                if isinstance(cursor, vertica_python.vertica.cursor.Cursor):
-                    with open("./VERTICAPY_DBSCAN_CLUSTERS_ID.csv", "r") as fs:
-                        cursor.copy(
-                            "COPY v_temp_schema.VERTICAPY_DBSCAN_CLUSTERS(node_id, cluster) FROM STDIN DELIMITER ',' ESCAPE AS '\\';",
-                            fs,
-                        )
+                drop_if_exists("v_temp_schema.{}".format(name_dbscan_clusters), method="table")
+                executeSQL("CREATE LOCAL TEMPORARY TABLE {}(node_id int, cluster int) ON COMMIT PRESERVE ROWS".format(name_dbscan_clusters), print_time_sql=False)
+                if isinstance(current_cursor(), vertica_python.vertica.cursor.Cursor):
+                    executeSQL("COPY v_temp_schema.{}(node_id, cluster) FROM STDIN DELIMITER ',' ESCAPE AS '\\';".format(name_dbscan_clusters), method="copy", print_time_sql=False, path="./{}.csv".format(name_dbscan_clusters))
                 else:
-                    cursor.execute(
-                        "COPY v_temp_schema.VERTICAPY_DBSCAN_CLUSTERS(node_id, cluster) FROM LOCAL './VERTICAPY_DBSCAN_CLUSTERS_ID.csv' DELIMITER ',' ESCAPE AS '\\';"
-                    )
+                    executeSQL("COPY v_temp_schema.{}(node_id, cluster) FROM LOCAL './{}.csv' DELIMITER ',' ESCAPE AS '\\';".format(name_dbscan_clusters, name_dbscan_clusters), print_time_sql=False)
                 try:
-                    cursor.execute("COMMIT")
+                    executeSQL("COMMIT", print_time_sql=False)
                 except:
                     pass
```

Comment:
`pass`?

Reply:
Done; The change will be made in the next commit.